src/commands/run.py

```diff
@@ -113,3 +123,22 @@ def remove_done_tasks():
             session.commit()
             Session.remove()
             run_print('Done')
+
+    def get_confirmation(configs, nodes):
+        click.echo("""
+You are about to run: {}\nOver nodes: {}\n
```

Comment:
Do no use `\n` in heredocs. It defeats the purpose of using the heredoc

Reply:
https://github.com/alces-software/adminware/pull/166/commits/06aa17a7cbe461cbf7d97fb60cd1d9a832ef5ac2 done here